modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/input_move_boxes.js

```diff
@@ -283,6 +290,10 @@ AUI.add(
 					var moveBtnRight = contentBox.one('.move-right');
 
 					var target = event.target;
+					var selectedOptions = target
+						.get('options')
+						.getDOMNodes()
+						.filter((option) => option.selected).length;
```

Comment:
I would expect `selectedOptions` variable to contain an array of options. We could add the `.length` later on, or rename it to something like `selectedOptionsCount`

Reply:
Yes, did this already